kiwi/tasks/result_bundle.py

```diff
@@ -123,45 +123,45 @@ def process(self):
                 bundle_file = ''.join(
                     [bundle_directory, '/', bundle_file_basename]
                 )
-                checksum_file = ''.join(
-                    [bundle_directory, '/', bundle_file_basename, '.sha256']
-                )
                 Command.run(
                     [
                         'cp', result_file.filename, bundle_file
                     ]
                 )
-                if result_file.compress:
+                if self.runtime_config.is_bundle_compression_requested() and \
+                   result_file.compress:
```

Comment:
Is it me or there is an strange indentation here? Anyway flake does not complain, so I guess this is _pythonic_ :smile:

Reply:
Yes it's strange but pythonic. The long line break in conditional statements always have the problem that the indented block has to belong to the condition. The indentation has to be in between of the condition and the block it belongs to if the condition line has a line break. Very long discussion about alternative style and pros and cons here: #861